frontend/packages/kubevirt-plugin/integration-tests/tests/models/kubevirtDetailView.ts

```diff
@@ -50,8 +53,12 @@ export class KubevirtDetailView extends DetailView {
 
   async removeDisk(name: string) {
     await this.navigateToTab(TABS.DISKS);
+    const count = await resourceRows.count();
     await kubevirtDetailView.selectKebabOption(name, 'Delete');
     await confirmAction();
+    do {
+      await browser.sleep(50);
```

Comment:
Why 50? Can we find a more reliable check than just a delay?

Reply:
Fixed.